tests/integration/test_http_post.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
```

Comment:
The Cachito source code is labeled as: `GPL-3.0-or-later`
Do we want to deviate the integration tests from it?

Reply:
yes, we want that (it's copy-paste mistake)